frontend/__tests__/components/buttons.test.tsx

```diff
@@ -0,0 +1,99 @@
+import { fireEvent, render } from '@testing-library/react';
+
+import { createRemixStub } from '@remix-run/testing';
+
+import { describe, expect, it, vi } from 'vitest';
+
+import { Button, ButtonLink } from '~/components/buttons';
+
+describe('Button Component', () => {
+  it('renders button with default props', () => {
+    const { getByText } = render(<Button>Click me</Button>);
+    const button = getByText('Click me');
+    expect(button).toBeInTheDocument();
+    expect(button.tagName).toBe('BUTTON');
+    expect(button).toHaveClass('inline-flex', 'items-center', 'justify-center', 'align-middle', 'font-medium', 'focus:outline-none', 'focus:ring-4');
```

Comment:
This assertion seems kind of fragile. Do we really care so much about the button styling that we need to check the css classes?

Reply:
Those classes are important for the button base css classes.

Reply:
The rendered button have more classes than that.

Reply:
🤷